### PR TITLE
fix #4 remove LFS

### LINKED ISSUE #4: test_data.fasttext.gz causes a git-lfs failure
When cloning the repo

```
git clone https://github.com/AgnieszkaFalenska/IMSnPars.git
```

git-lfs fails:
```
...
git-lfs filter-process: git-lfs: command not found
fatal: the remote end hung up unexpectedly
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

The cause is `test_data.fasttext.gz` because the file is too big 
- Rule-of-thumb: Never store data files in git repositories (Only these tiny statistics datasets might be ok, or shrinked test files for unit tests).
- Data files (e.g. serialized modes, embedding files, the actual training set) must be loaded from another server, e.g. when you call `stanza.download` the data files are loaded from a file server. These files are not inside the git repo.

@AgnieszkaFalenska This is a "bug" labelled issue

### PR DESCRIPTION
**Pull request by ulf1**
- [x] (large) data files deleted
- [x] git-lfs removed
- [x] downloader script for (large) test data files
- [x] setup.py and requirements files for pkg conflict resolving

@AgnieszkaFalenska Don't merge until the code is tested

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,15 +38,24 @@ Later it was extended with multi-task training for the paper [Integrating Graph-
...
+# install virtual env 
+python3.6 -m venv .venv
+source .venv/bin/activate
+pip install --upgrade pip
 
-> [NetworkX package](https://networkx.github.io/)
+# install imsnpars package
+pip install -r requirements.txt --use-feature=2020-resolver
+pip install -r requirements-dev.txt --use-feature=2020-resolver
+python setup.py develop -q
```
Comment (ulf1) [inline README.md:55]: Should be `python setup.py install`

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# neural network model
+dynet>=2.0.0
+networkx<2.5,>=2.1
+
+# other packages
```
Comment (ulf1) [inline requirements.txt:5]: What packages are missing?

File: scripts/imsnpars_downloader.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import os
+from pathlib import Path
+import requests
+
+
+if __name__ == '__main__':
+    # TODO: IMSnPars test files
+    PATH = f"{str(Path.home())}/imsnpars_data/systests"
+    os.makedirs(PATH, exist_ok=True)
+    URL = "http://???.de/pfad/test_data.fasttext-v123456789.gz"
```
Comment (ulf1) [inline scripts/imsnpars_downloader.py]: @AgnieszkaFalenska Can Uni-Stuttgart provide a file server (or object store) for public downloads?

File: scripts/imsnpars_downloader.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    os.system(f"wget -O '{PATH}/test_data.fasttext.gz' '{URL}'")
+    URL = "http://???.de/pfad/test_data.elmo.ly-2.hdf5.gz"
```
Comment (ulf1) [inline scripts/imsnpars_downloader.py]: Download endpoint is missing

File: scripts/imsnpars_downloader.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    os.system(f"wget -O '{PATH}/test_data.elmo.ly-2.hdf5' '{URL}'")
+
+    # TODO: IMSnPars folder with serialized model
+    PATH = f"{str(Path.home())}/imsnpars_data/models"
+    os.makedirs(PATH, exist_ok=True)
+    URL = "http://???.de/pfad/model-trans-20201130/"
```
Comment (ulf1) [inline scripts/imsnpars_downloader.py]: Download endpoint is missing

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+      author='Agnieszka Faleńska',
+      author_email='agnieszka.falenska@ims.uni-stuttgart.de',
+      license='Apache License 2.0',
+      packages=['imsnpars'],
+      install_requires=[
+          'dynet>=2.0.0',
+          'networkx<2.5,>=2.1'
+      ],
+      scripts=[
+          "scripts/download.py"
```
Comment (ulf1) [inline setup.py]: rename to `imsnpars_downloader.py`

File: scripts/imsnpars_downloader.py (hunk 3 above)
Comment (AgnieszkaFalenska) [reply scripts/imsnpars_downloader.py]: Yes, I asked for a designated place (also for pre-trained models).

**Comment by AgnieszkaFalenska** [comment]
Hi Ulf, the files are available here:
https://www2.ims.uni-stuttgart.de/htdocs/data/imsnpars/test_data.fasttext.gz
https://www2.ims.uni-stuttgart.de/data/imsnpars/test_data.elmo.ly-2.hdf5

File: README.md (hunk 1 above)
Comment (ulf1) [reply README.md:55]: added as issue

**Comment by ulf1** [comment]
Hello @AgnieszkaFalenska 
could you take a look? 

1. Github Actions must be activated in the repo's "Settings" section. The yml tests if installation works, and system tests are running. See https://github.com/AgnieszkaFalenska/IMSnPars/pull/5/files#diff-4fa5cb70d2204c18dfad1fb77e16b837e36d00163d85445f30be16e60ec578d4
2. The Downloader script `imsnpars_downloader.py` can also download the HDT dataset (See https://github.com/AgnieszkaFalenska/IMSnPars/pull/5/files#diff-605f6cd5e834549289662f887558d60b1481dc7227dc373e3a230f84078ab5e7) for demo purposes (see Readme.md https://github.com/AgnieszkaFalenska/IMSnPars/pull/5/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)
3. The setup.py looks ready for PyPi (see https://github.com/AgnieszkaFalenska/IMSnPars/pull/5/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7)

**Pull request merged**